src/main/java/com/example/parkingapp/domain/User.java

```diff
@@ -0,0 +1,120 @@
+package com.example.parkingapp.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
```

Comment:
importy z * są niemile widziane na review

Reply:
Domyślne ustawienia IntelliJ, ale rozumiem dlaczego to problem (nie widać które dokładnie klasy są używane)